Sources/Event.swift

```diff
@@ -861,6 +861,63 @@ extension Signal.Event {
 			}
 		}
 	}
+	
+	internal static func collect(every interval: DispatchTimeInterval, on scheduler: DateScheduler, skipEmpty: Bool, discardsWhenCompleted: Bool) -> Transformation<[Value], Error> {
```

Comment:
Oh, let's make this `discardWhenCompleted` (no _s_) since we're doing that in #605 as well.